api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/VerificationResult.java

```diff
@@ -36,7 +36,8 @@
 
     @JsonIgnore
     public boolean isAddressMismatch() {
-        return VerificationOutcome.NOT_MATCHED == addressLine1Match || VerificationOutcome.NOT_MATCHED == postcodeMatch;
+        return EligibilityOutcome.CONFIRMED == eligibilityOutcome
+                && (VerificationOutcome.NOT_MATCHED == addressLine1Match || VerificationOutcome.NOT_MATCHED == postcodeMatch);
```

Comment:
I checked with Karen, and she wanted to include a check that we would only send an email & letter for an address mismatch if the claimant was otherwise eligible.

Reply:
I think this is a redundant check though because if the EligibilityOutcome is not CONFIRMED then these values will always be NOT_SET, I think this should be removed as it confuses the basis of how the VerificationResult is built.